### PR TITLE
Fixed 02_pong_a2c.py and speed up the convergence/process

### PR DESCRIPTION
**Pull request by JeremieGauthier**
I changed the line adv_v = vals_ref_v - value_v.detach() to adv_v = vals_ref_v - value_v.squeeze(-1).detach(). It seems the convergence is much faster. According to the A2C algorithm, it is just logic to apply Q(a, s) - V(s) where Q(a, s) and V(s) with the same shape.

The call to detach() is important here as we don't want to propagate the PG into our value approximation head.

I added np.array() in two places to speed up the process

### CODE REVIEW

**Comment by Shmuma** [comment]
Thanks a lot!
